pyglmnet/pyglmnet.py

```diff
@@ -881,7 +863,7 @@ def score(self, X, y):
 
 
 class GLMCV(object):
-    """Class for estimating regularized generalized linear models (GLM)
+    r"""Class for estimating regularized generalized linear models (GLM)
```

Comment:
why do you need this? It's not related to python 2, is it?

Reply:
Running tests gave warnings [like this](https://github.com/pallets/jinja/issues/646). It's unrelated to Python 2.